### ISSUE TITLE
Use consistent text for 'No results'

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#67728]
### Description

In various packages in the editor, when some search or filtering doesn't return results, some text is shown to users to inform there are no results. This text is inconsistent across the editor. Searching on current trunk:

4 occurrences of
`__( 'No results.' )`
with a trailing period.

3 occurrences of
`__( 'No results' )`
with no trailing period

7 occurrences 
`__( 'No results found.' )`
with a trailing period.

2 occurrences of
`__( 'No results found' )`
with no trailing period.

I'm not sure whether there's a specific design reason to use different text but I'd tend to think the editor should always use the same message everywhere.

### Step-by-step reproduction instructions

N/A
Search the codebase for the stringe above.

### Please confirm that you have searched existing issues in the repo.

- [x] Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

- [x] Yes

### Please confirm which theme type you used for testing.

- [ ] Block
- [ ] Classic
- [ ] Hybrid (e.g. classic with theme.json)
- [ ] Not sure

**Comment by Rishit30G** [comment]
I agree with the same thing, as and when we get confirmation on the above, I would like to work on this issue 
Thank you! 🙇🏻
